libs/api/domains/cms/src/lib/models/organizationPage.model.ts

```diff
@@ -38,6 +38,9 @@ export class OrganizationPage {
   @Field(() => [LinkGroup])
   menuLinks!: Array<LinkGroup>
 
+  @Field(() => LinkGroup, { nullable: true })
+  secondaryMenu!: LinkGroup | null
+
```

Comment:
If this is not exclusively external links we should consider making this an actual menu to support reference links for dynamic link resolution

Reply:
They are mostly internal links to organization subpages, those have two slugs `/stofnanir/[pageSlug]/[subpageSlug]`. We need to change Reference Links or the UI extension to make that work well. We're gonna do that in a separate PR as we discussed :)

Reply:
Yes, supporting this type in ReferenceLink would be nice for consistency until we finish the UI extension work.
We should look into that ASAP in a separate PR, shouldn't be to complicated since OrganizationPage already includes the parent ref, could do similar logic to typeMap but with custom link resolvers to support special cases